packages/components/tooltip/README.mdx

```diff
@@ -0,0 +1,54 @@
+# @contentful/f36-tooltip
+
+---
+
+title: 'Tooltip'
+type: 'component'
+status: 'stable'
+slug: /components/tooltip/
+github: 'https://github.com/contentful/forma-36/tree/master/packages/components/tooltip'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-tooltip--default'
+
```

Comment:
Please, add an automatically generated props table. We have an example of it in all other migrated components.